SocialCareCaseViewerApi/V1/Boundary/Requests/UpdateMashReferral.cs

```diff
@@ -0,0 +1,30 @@
+
+using FluentValidation;
+
+#nullable enable
+namespace SocialCareCaseViewerApi.V1.Boundary.Requests
+{
+    public class UpdateMashReferral
+    {
+        public int WorkerId { get; set; }
+        public string UpdateType { get; set; } = null!;
+        public string? Decision { get; set; }
+        public bool? RequiresUrgentContact { get; set; }
+    }
+
+    public class UpdateMashReferralValidator : AbstractValidator<UpdateMashReferral>
+    {
+        public UpdateMashReferralValidator()
+        {
+            RuleFor(x => x.Decision)
+                .NotNull().When(x => x.UpdateType.ToUpper() == "SCREENING-DECISION")
+                .WithMessage("Must provide a decision");
+            RuleFor(x => x.Decision)
+                .MinimumLength(1).When(x => x.UpdateType.ToUpper() == "SCREENING-DECISION")
+                .WithMessage("Must provide a decision");
```

Comment:
out of curiosity, what is the difference between a null value and a value that has a <1 length? isn't the latter null by definition? 🤔

Reply:
null is to represent nothing being sent, an empty string ("") is not null but would also not be something we want to accept.